app/views/comments/new.erb

```diff
@@ -0,0 +1,5 @@
+<form id="new-comment" method="POST" action="/suggestions">
+  <input class="text" type="text" name="comment[comment_text]" placeholder="Comment">
```

Comment:
Comment create should hit either `/comments` or `/suggestions/<%= suggestion.id %>/comments`